Framework/include/BJet.h

```diff
@@ -70,7 +70,7 @@ class BJet
         for (unsigned int ijet = 0; ijet < Jets.size(); ++ijet)
         {
             utility::LorentzVector lv = Jets.at(ijet);
-            double bdisc = Jets_bJetTagDeepCSVtotb.at(ijet);
+            double bdisc = Jets_bJetTagDeepFlavourtotb.at(ijet);
```

Comment:
This is all we need to do in Framework to use DeepFlavour instead of DeepCSV right? With the exception of the TopTagger code

Reply:
Yeah, this is the only meaningful reference in `Framework` besides trivially `PrepNtupleVars` and `RunTopTagger`. I was also now just looking to get rid of the DeepCSV components from `MakeMVAVariables` as they are very deprecated there at this point.

Reply:
Done in 389bb78

Reply:
I approve this idea. Feel free to merge this PR in if you want to work on top of it.